lib/task/import/csvAuthorityRecordImportTask.class.php

```diff
@@ -284,7 +284,14 @@ public function execute($arguments = [], $options = [])
                                 continue;
                             }
 
-                            if (null !== $relation = QubitActor::setTermRelationByName($places[$i], $options = ['taxonomyId' => QubitTaxonomy::PLACE_ID, 'culture' => $self->columnValue('culture')])) {
+                            $relation = QubitActor::setTermRelationByName(
+                                            $places[$i],
+                                            ['taxonomyId' => QubitTaxonomy::PLACE_ID,
+                                                'culture' => $self->columnValue('culture'), ]);
+
+                            $relationAlreadyExists = QubitFlatfileImport::objectTermRelationExists($self->object->id, $relation->termId);
```

Comment:
I think `$relation->term` will throw a fatal error if `$relation === null`, which can happen if a term doesn't exist and the current user doesn't have permission to create a new term in the taxonomy.

Reply:
Good catch!